frb_rust/src/support.rs

```diff
@@ -42,11 +41,37 @@ pub unsafe fn box_from_leak_ptr<T>(ptr: *mut T) -> Box<T> {
     Box::from_raw(ptr)
 }
 
+mod sealed {
+    pub trait Sealed {}
+    impl Sealed for i64 {}
+    impl Sealed for u64 {}
+}
```

Comment:
nit: Are we using it somewhere, or should it be deleted

Reply:
To be deleted.